iFixit/react-commerce
# Optimize images

This PR closes #51 and #47 and fixes a few minor issues.

## QA

1. Visit the Vercel PR preview
2. Visit `/collections/macbook`
3. Verify that products without image render a placeholder image
4. Verify that images are rendered in a 4:3 ratio so that they are displayed correctly (some images though don't have a 4:3 ratio therefore they will be cropped)
